command/agent/cache/lease_cache.go

```diff
@@ -373,15 +373,6 @@ func (c *LeaseCache) startRenewing(ctx context.Context, index *cachememdb.Index,
 			}
 			c.logger.Debug("renewal halted; evicting from cache", "path", req.Request.URL.Path)
 			return
-		case renewal := <-renewer.RenewCh():
-			// This case captures secret renewals. Renewed secret is updated in
-			// the cached index.
-			c.logger.Debug("renewal received; updating cache", "path", req.Request.URL.Path)
```

Comment:
I think we should at least log that renewal has occurred, even if no other action is taken.